src/modules/ttlmonitor/TTLMonitorInstrument.cpp

```diff
@@ -49,9 +50,14 @@ void TTLMonitorInstrument::processMonitorReadouts(void) {
   // possible, but rings and fens
   // could still be outside the configured range, also
   // illegal time intervals can be detected here
-  assert(Serializer != nullptr);
-  Serializer->pulseTime(ESSReadoutParser.Packet.Time
+
+  
+  //TODO, have proper assertion heres
+  //assert(Serializers != nullptr);
+  for (EV44Serializer &Serializer : Serializers){
```

Comment:
store counters inside serializer - count different kinds of flushes
store timer inside serializer